viewer/src/datamodels/cad/ModelRenderAppearance.ts

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+export type NodeFrontRenderDelegate = (treeIndex: number) => boolean;
+export type NodeBorderDelegate = (treeIndex: number) => NodeBorderMetadata;
+
+export type NodeBorderMetadata = {
+  color: [number, number, number, number];
+  width: number;
+};
+
+export interface ModelRenderAppearance {
+  renderInFront?: NodeFrontRenderDelegate;
+  renderBorder?: NodeBorderDelegate;
+}

```

Comment:
I think this should be merged with `ModelNodeApperance`. Also (but not related to this PR), `GlobalNodeApperance` needs to die.